lib/phoenix_live_view/controller.ex

```diff
@@ -48,9 +48,12 @@ defmodule Phoenix.LiveView.Controller do
         )
 
       {:stop, %Socket{redirected: {:redirect, opts}} = socket} ->
```

Comment:
Match on status here, remove put_status, and use Map.delete instead of drop. :)

Reply:
Thanks! Addressed - is this what you had in mind?